src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,58 @@
+import SideBarItem from "./SideBarItem";
+import { Box, Flex } from "@chakra-ui/react";
+// import './style.css';
+
+const SideBar = () => {
+  const sideBarContent = [
```

Comment:
You can use svg and[ Image](https://chakra-ui.com/docs/components/media-and-icons/image) here
<img width="1402" alt="image" src="https://user-images.githubusercontent.com/61776449/168297592-68b473ae-4bfc-4e6f-9d1d-ef24486b48ba.png">

Reply:
Create a new folder called assets in src, then create a folder called svg in assets, and put the svg icon in this folder